### DISCUSSION TITLE
Help Required: Unable to build and run project using Xcode

### DISCUSSION THREAD

**Discussion by SwapnanilDhol** [Shopify/tophat#7] [Q&A] [answered]
Hello!

I was attempting to build the .xcproj file using Xcode 15.2 but when the application attempted to run I got to this fatalError and I'm not sure where to find the `ios-deploy` package.

```
fatalError("Attempting to execute ios-deploy but it was not found in the application bundle")
```

![Image](https://github.com/user-attachments/assets/d75f196b-c060-4c85-be1b-e5a8984a0437)

**Comment by lfroms** [comment, marked as answer]
Hi @SwapnanilDhol, you need to sync the Git submodules as described in the [contributing guide](https://github.com/Shopify/tophat/blob/main/CONTRIBUTING.md#developing).  The `ios-deploy` project is made available through a submodule located in the `Vendor/ios-deploy` directory.  Have you tried that?

**Comment by SwapnanilDhol** [comment]
Hi @lfroms. Thank you very much. I think I missed running the sub-modules step. It works now!
